app/assets/javascripts/discourse/components/learn-more.js.es6

```diff
@@ -0,0 +1,23 @@
+export default Ember.Component.extend({
+  type: null,
+
+  uploadInfo(type){
+    $.ajax({
```

Comment:
We wrapped jQuery's ajax call in `Discourse.ajax` which is now the recommended way of talking to the API